.ci/metrics/metrics.py

```diff
@@ -12,9 +13,29 @@
 GRAFANA_URL = (
     "https://influx-prod-13-prod-us-east-0.grafana.net/api/v1/push/influx/write"
 )
-GITHUB_PROJECT = "llvm/llvm-project"
-WORKFLOWS_TO_TRACK = ["LLVM Premerge Checks"]
-SCRAPE_INTERVAL_SECONDS = 5 * 60
+SCRAPE_INTERVAL_SECONDS = 60
```

Comment:
I'm assuming you've tested for this one hour plus to ensure we don't hit any API rate limits?

Reply:
Forgot to revert this, used 60s locally for testing, but 5mn is enough for what we need.